<launch>
    <group ns="robot_0">
        <node pkg="micros_swarm_framework" type="micros_swarm_framework_app2_node" name="micros_swarm_framework_app2_node0" output="screen">
            <param name="unique_robot_id" value="0"/>
        </node>
    </group>
    <group ns="robot_1">
        <node pkg="micros_swarm_framework" type="micros_swarm_framework_app2_node" name="micros_swarm_framework_app2_node1" output="screen">
            <param name="unique_robot_id" value="1"/>
        </node>
    </group>
    <group ns="robot_2">
        <node pkg="micros_swarm_framework" type="micros_swarm_framework_app2_node" name="micros_swarm_framework_app2_node2" output="screen">
            <param name="unique_robot_id" value="2"/>
        </node>
    </group>
    <group ns="robot_3">
        <node pkg="micros_swarm_framework" type="micros_swarm_framework_app2_node" name="micros_swarm_framework_app2_node3" output="screen">
            <param name="unique_robot_id" value="3"/>
        </node>
    </group>
    <group ns="robot_4">
        <node pkg="micros_swarm_framework" type="micros_swarm_framework_app2_node" name="micros_swarm_framework_app2_node4" output="screen">
            <param name="unique_robot_id" value="4"/>
        </node>
    </group>
    
    
    
    <group ns="robot_5">
        <node pkg="micros_swarm_framework" type="micros_swarm_framework_app2_node" name="micros_swarm_framework_app2_node5" output="screen">
            <param name="unique_robot_id" value="5"/>
        </node>
    </group>
    <group ns="robot_6">
        <node pkg="micros_swarm_framework" type="micros_swarm_framework_app2_node" name="micros_swarm_framework_app2_node6" output="screen">
            <param name="unique_robot_id" value="6"/>
        </node>
    </group>
    <group ns="robot_7">
        <node pkg="micros_swarm_framework" type="micros_swarm_framework_app2_node" name="micros_swarm_framework_app2_node7" output="screen">
            <param name="unique_robot_id" value="7"/>
        </node>
    </group>
    <group ns="robot_8">
        <node pkg="micros_swarm_framework" type="micros_swarm_framework_app2_node" name="micros_swarm_framework_app2_node8" output="screen">
            <param name="unique_robot_id" value="8"/>
        </node>
    </group>
    <group ns="robot_9">
        <node pkg="micros_swarm_framework" type="micros_swarm_framework_app2_node" name="micros_swarm_framework_app2_node9" output="screen">
            <param name="unique_robot_id" value="9"/>
        </node>
    </group>
    
    
    
    <group ns="robot_10">
        <node pkg="micros_swarm_framework" type="micros_swarm_framework_app2_node" name="micros_swarm_framework_app2_node10" output="screen">
            <param name="unique_robot_id" value="10"/>
        </node>
    </group>
    <group ns="robot_11">
        <node pkg="micros_swarm_framework" type="micros_swarm_framework_app2_node" name="micros_swarm_framework_app2_node11" output="screen">
            <param name="unique_robot_id" value="11"/>
        </node>
    </group>
    <group ns="robot_12">
        <node pkg="micros_swarm_framework" type="micros_swarm_framework_app2_node" name="micros_swarm_framework_app2_node12" output="screen">
            <param name="unique_robot_id" value="12"/>
        </node>
    </group>
    <group ns="robot_13">
        <node pkg="micros_swarm_framework" type="micros_swarm_framework_app2_node" name="micros_swarm_framework_app2_node13" output="screen">
            <param name="unique_robot_id" value="13"/>
        </node>
    </group>
    <group ns="robot_14">
        <node pkg="micros_swarm_framework" type="micros_swarm_framework_app2_node" name="micros_swarm_framework_app2_node14" output="screen">
            <param name="unique_robot_id" value="14"/>
        </node>
    </group>
    
    
    
    <group ns="robot_15">
        <node pkg="micros_swarm_framework" type="micros_swarm_framework_app2_node" name="micros_swarm_framework_app2_node15" output="screen">
            <param name="unique_robot_id" value="15"/>
        </node>
    </group>
    <group ns="robot_16">
        <node pkg="micros_swarm_framework" type="micros_swarm_framework_app2_node" name="micros_swarm_framework_app2_node16" output="screen">
            <param name="unique_robot_id" value="16"/>
        </node>
    </group>
    <group ns="robot_17">
        <node pkg="micros_swarm_framework" type="micros_swarm_framework_app2_node" name="micros_swarm_framework_app2_node17" output="screen">
            <param name="unique_robot_id" value="17"/>
        </node>
    </group>
    <group ns="robot_18">
        <node pkg="micros_swarm_framework" type="micros_swarm_framework_app2_node" name="micros_swarm_framework_app2_node18" output="screen">
            <param name="unique_robot_id" value="18"/>
        </node>
    </group>
    <group ns="robot_19">
        <node pkg="micros_swarm_framework" type="micros_swarm_framework_app2_node" name="micros_swarm_framework_app2_node19" output="screen">
            <param name="unique_robot_id" value="19"/>
        </node>
    </group>
    
</launch>
